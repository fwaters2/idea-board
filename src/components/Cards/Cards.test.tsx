import React from "react";
import { Cards } from ".";
import { render } from "@testing-library/react";

describe("Cards component", () => {
  const props = {
    children: [<div key={1}>1</div>, <div key={2}>2</div>],
    handleDeleteCard: () => {},
    handleUpdateCard: () => {},
  };
  test("should render correctly", () => {
    const { container } = render(<Cards {...props} />);
    expect(container).toMatchSnapshot();
  });
});
