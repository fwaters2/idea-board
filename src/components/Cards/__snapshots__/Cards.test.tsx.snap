// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards component should render correctly 1`] = `
<div>
  <ul
    class="cards"
  >
    <li
      style="opacity: 0; transform: translateX(-500px) translateZ(0);"
    >
      <div>
        1
      </div>
    </li>
    <li
      style="opacity: 0; transform: translateX(-500px) translateZ(0);"
    >
      <div>
        2
      </div>
    </li>
  </ul>
</div>
`;
